README.md

```diff
@@ -223,11 +223,11 @@ created to house a collection of items in plural form and using camelCase (eg: p
 ## File naming/structure
 Files should be named after the component/function/constants they export, respecting the casing used for it. ie:
 
-- If you export a constant named `CONST` it's file/directory should be named the `CONST`.
+- If you export a constant named `CONST` its file/directory should be named the `CONST`.
 - If you export a component named `Text` the file/directory should be named `Text`.
 - If you export a function named `guid` the file/directory should be named `guid`.
 - For files that are utilities that export several functions/classes use the UpperCamelCase version ie: `DateUtils`.
-- HOCs should be named in camelCase like withOnyx.
+- HOCs should be named in camelCase like `withOnyx`.
```

Comment:
Not gonna lie, I had no idea what an HOC was when I first got to Expensify, especially since I was using functional components and hooks (which I assume to be the norm with React developers these days). 

Any chance we can link to some docs + an example in our code here? Also, perhaps spell out `Higher-Order Components` the first time it's referenced.

Reply:
Good idea - I think we could just link to [this page](https://reactjs.org/docs/higher-order-components.html) for "what are HOCs", and maybe this whole point could look like:

> - [Higher-Order Components](https://reactjs.org/docs/higher-order-components.html) (HOCs) should be named in camelCase, like `withOnyx` (example [here](https://github.com/Expensify/App/blob/08a9b226a27a6f24bb11c3006cf433739c834860/src/Expensify.js#L214-L231)).

Reply:
Hmm on second thought I don't think we need the `(example here)` part, if we call out `withOnyx` it should be pretty easy to search for examples right?

Reply:
I dig!!